src/booleans.js

```diff
@@ -1,61 +1,127 @@
 function negate(a) {
-  // your code here
+  if(a === true){
+    return false;
+  } else if(a === false){
+    return true;
+  }
 };
 
 function both(a, b) {
-  // your code here
+ if ((a === true) && (b === true)){
+   return true;
+ } else {
+   return false;
+ }
 };
 
 function either(a, b) {
-  // your code here
+  if ((a === true) || (b === true)){
```

Comment:
Same here, you could remove the if statement and also because the tests are always passing in booleans, you could even simplify to checking the truthiness of both sides. eg return a || b